assets/dev/scss/frontend/widgets/spacer.scss

```diff
@@ -5,14 +5,11 @@
 	}
 }
 
-.e-con {
-	--container-widget-width: 100%;
-}
-
 .e-con,
 .e-con-inner {
 
 	& > .elementor-widget-spacer {
+		--container-widget-width: 100%;
 		width: var( --container-widget-width, var( --spacer-size ) );
 		--align-self: var( --container-widget-align-self, initial );
 		--flex-shrink: 0;
```

Comment:
One small comment.
For the readibility of the code it might nice to first display all css variables and only afterwards the normal css selector.

Reply:
@hein-obox Done.